mlos_bench/mlos_bench/tests/environments/local/local_env_stdout_test.py

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Unit tests for extracting data from LocalEnv stdout.
+"""
+
+from mlos_bench.tunables.tunable_groups import TunableGroups
+from mlos_bench.tests.environments.local import create_local_env, check_local_env_success
+
+
+def test_local_env_stdout(tunable_groups: TunableGroups) -> None:
+    """
+    Print benchmark results to stdout and capture them in the LocalEnv.
+    """
+    local_env = create_local_env(tunable_groups, {
+        "run": [
+            "echo 'Benchmark results:'",  # This line should be ignored
+            "echo 'latency,111'",
+            "echo 'throughput,222'",
+            "echo 'score,0.999'",
+        ],
+        "parse_results_stdout": r"(\w+),([0-9.]+)",
```

Comment:
Ah, this is actually more like a grep pattern, not parse/extract.  Is that correct?

Reply:
When you called it parse, I was thinking of it more like a `sed`

Reply:
yes, it's like grep in a sense that we can use it to match only some specific parts of the output, but then it must have two groups to extract the metric id and the value. This way we can discard large portions of stdout that do not contain the data we want to extract

Reply:
Ack, makes more sense now with the example you posted above.  Thanks!